Gopkg.toml

```diff
@@ -98,17 +98,9 @@ required = [
   name = "gopkg.in/yaml.v2"
   version = "2.0.0"
 
-[[constraint]]
-  name = "k8s.io/api"
-  version = "kubernetes-1.10.0"
-
-[[constraint]]
-  name = "k8s.io/apimachinery"
-  version = "kubernetes-1.10.0"
-
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "~7.0.0"
+  version = "v9.0.0"
```

Comment:
@jackfrancis These are where I have manually typed in changes. The rest are all auto-generated.